### ISSUE TITLE
Posetools v1.1.1 Issue Report

### ISSUE THREAD

**Issue by incognisame** [IllusionMods/KK_Plugins#125]
![Righthand](https://user-images.githubusercontent.com/100230290/155208460-9ec6379f-395b-4c49-85eb-4cc0c3a21e3a.png)

Hello Devs.

1.When I load the KK, KKS pose png files on AI/HS2, the rotation values of the fingers of the right hand don't seem to be copied correctly.
On the other hand, the fingers of the left hand appear to have been copied normally.

If FK is turned on and then FK of the right hand is also turned on (in KK), the Right hand's fingers are twisted by default for some reason.
So it could be related to this issue.

Also, even if the dat/png file created in AI/HS2 is loaded in KK, the fingers of the right hand are twisted.

2.PH's pose dat files aren't compatible with other illusion games for some reason.
In other words, PH pose dat files can't be loaded in other illusion games, and pose dat files of other illusion games can't be loaded in PH.
(One of the workarounds is to load a desired pose in HS, save it as a scene file, load it in PH Studio, and then save the pose.)
Anyway, Even with the Posetools, PH seems to be incompatible with other illusion games's pose PNG files.

Have a nice day.

**Comment by incognisame** [comment]
After writing this issue and thinking about it, I think it may be a problem originating from the illusion game itself rather than a plugin issue, but still

**Comment by DeathWeasel1337** [comment]
Indeed, these are issues with the game itself. The only thing that allows us to load poses between games is the fact that Illusion hasn't updated the pose save and load code since HS1 (PH poses fail to load in areas outside of the pose code). The fingers being twisted is something that I have fixed as part of IllusionFixes, if you hold ctrl and load a pose it will correct the fingers. Now that ExtSave exists for poses, it saves the name of the game to the pose and it should be possible to fix this automatically for any poses created since BepisPlugins r17. I do plan to implement this at some point.

**Comment by incognisame** [comment]
I understand. Thanks for letting me know the workaround.(Ctrl)
Also, I am glad and appreciate that it can be automatically fixed in the future.

**Comment by DeathWeasel1337** [comment]
I released a new IllusionFixes with the fix for automatically correcting finger rotation for KK poses in AI/HS2
https://github.com/IllusionMods/IllusionFixes/releases/tag/v19.2

**Comment by incognisame** [comment]
Stellar job. Thank you.
